src/args/args.go

```diff
@@ -22,4 +22,5 @@ type ArgumentList struct {
 	Vhosts               string `default:"" help:"JSON array of vhost names from which to collect metrics."`
 	VhostsRegexes        string `default:"" help:"JSON array of vhost name regexes from which to collect metrics."`
 	ShowVersion          bool   `default:"false" help:"Print build information and exit"`
+	Timeout              int    `default:"30" help:"Timeout in seconds to timeout the connection to RabbitMQ endpoint."`
```

Comment:
Nit: why not using something like:

```Go
Timeout time.Duration `default:"30s" ...`

Reply:
You are right and I would like to have a measurement unit with the metrics but we will have two issues:
 * Arguments are parsed by the integration SDK and have no possibility to hook to any parser/unmarshaller so I would have to pass a string to where we need the timeout and parse there. So the error will be hidden in the code.
 * We already have this UX with other integrations like in [mssql](https://github.com/newrelic/nri-mssql/blob/master/src/args/argument_list.go#L25) or [postgres](https://github.com/newrelic/nri-postgresql/blob/master/src/args/argument_list.go#L24).
 
 I said "you are right" because we had issues in the past for not having units so people thought that [Kafka timeout](https://github.com/newrelic/nri-kafka/blob/master/src/args/args.go) was on seconds while it is in milliseconds.
 
 On the other hand, I would not like to start a new UX at this point.